maintenance/wikia/deleteLocalGroup.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+require_once( __DIR__ . '/../Maintenance.php' );
+
+/**
+ * Use this script together with "run_maintenance" to delete all user assignments to the given local group
+ */
+class DeleteLocalGroupMaintenance extends Maintenance {
+
+	const TABLE_NAME = 'user_groups';
+
+	static $allowed_groups = [
+		'upwizcampeditors'
+	];
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = 'Deletes a given local group from all wikis';
+		$this->addOption( 'group', 'Name of group to delete', true, true );
+		$this->addOption( 'delete', 'Do actually delete the data (no dry run mode)', false, false );
+	}
+
+	public function execute() {
+		global $wgDBname, $wgDBCluster;
+
+		$group = $this->getOption( "group" );
+		$delete = $this->getOption( "delete", false );
+		if ( !in_array( $group, self::$allowed_groups ) ) {
+			die( "Error: requested group " . $group . " not in the allowed white-list\n" );
+		}
+
+		$then = microtime( true );
+		if ( !$delete ) {
+			$this->output( "Getting entries count for group on {$wgDBname} ({$wgDBCluster})\n" );
+			$dbw = $this->getDB( DB_SLAVE );
+
+			$res = $dbw->select(
```

Comment:
use `selectField` which returns a single scalar value so you don't need the weird expression:
`->fetchRow()[0]` a few lines below